languages/javascript/concepts/strings/about.md

```diff
@@ -1 +1,5 @@
-TODO: add information on strings concept
+The key thing to remember about JavaScript strings is that we can call methods on them like `toUpperCase`, `trim`, etc.
```

Comment:
Missing a lot of information on strings (currently in introduction)

Reply:
I've added the information from the introduction.